services/gtfs-rt-archiver-v3/docker-compose.yml

```diff
@@ -8,7 +8,8 @@ x-gtfs-rt-archiver-v3-common:
     CALITP_HUEY_REDIS_HOST: redis
     GOOGLE_APPLICATION_CREDENTIALS: /tmp/secret.json
     CALITP_BUCKET__AIRTABLE: "gs://test-calitp-airtable"
-    CALITP_BUCKET__GTFS_RT_RAW: "gs://test-calitp-gtfs-rt-raw"
+    CALITP_BUCKET__GTFS_CONFIG: "gs://test-calitp-gtfs-download-config"
+    CALITP_BUCKET__GTFS_RT_RAW: "gs://dev-calitp-gtfs-rt-raw"
```

Comment:
when would this be bumped to `-v2`?

Reply:
For local running in docker-compose, never; since we have a live "test" version of this, I realized we should be using a `dev` bucket when running locally, for the raw data.

Reply:
ahhh ok, I was not following that `test`/`dev` distinction (which I do think should be a larger conversation in general re: what is the purpose of the "live test")